### PR TITLE
PIPE2D-921: Add class FluxModelSet

### PR DESCRIPTION
**Pull request by SogoMineo**
FluxModelSet is a class that helps to deal with spectra
in fluxmodeldata package.

### CODE REVIEW

File: python/pfs/drp/stella/fluxModelSet.py [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+
+        Elements of this list also contain broadband photometries
+        that are integrations of the synthetic spectra
+        with various filters applied to them.
+
+        Returns
+        -------
+        parameters : `numpy.array`
+            A structured array,
+            whose columns are SED parameters and various broadband fluxes.
```
Comment (PaulPrice) [inline python/pfs/drp/stella/fluxModelSet.py:43]: List the parameters?

File: tests/test_fluxModelSet.py [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+@unittest.skipIf(dataDir is None, "fluxmodeldata not setup")
+class FluxModelSetTestCase(lsst.utils.tests.TestCase):
+    def setUp(self):
+        self.fluxModelSet = FluxModelSet(os.path.join(dataDir))
+
+    def testGetSpectrum(self):
+        """Test getFileName()
+        """
+        parameters = self.fluxModelSet.parameters
```
Comment (PaulPrice) [inline tests/test_fluxModelSet.py:24]: Check that this contains the expected list of parameters?

**Review by PaulPrice — APPROVED** [review]
Awesome!

Made some suggestions for you to adopt or ignore as appropriate.

File: tests/test_fluxModelSet.py (hunk 2 above)
Comment (SogoMineo) [reply tests/test_fluxModelSet.py:23]: If any of the expected parameters ("teff", "logg", "m", "alpha") do not exist, an exception will be raised from below, which makes the test fail. I think this is already sufficient.

**Pull request merged**